packages/Algernon.package/ALGCore.class/instance/handlesMouseMove..st

```diff
@@ -0,0 +1,3 @@
+initialization
+handlesMouseMove: evt
```

Comment:
Why do we need to handle mouse move? Can't we use mouse hover?

Reply:
I did not find a hover method/event.

Reply:
You can use `Morph>>mouseEnter:` and `Morph>>mouseLeave:`